datafusion/functions-array/src/resize.rs

```diff
@@ -85,7 +86,9 @@ impl ScalarUDFImpl for ArrayResize {
 }
 
 /// array_resize SQL function
-pub fn array_resize_inner(arg: &[ArrayRef]) -> datafusion_common::Result<ArrayRef> {
+pub(crate) fn array_resize_inner(
+    arg: &[ArrayRef],
+) -> datafusion_common::Result<ArrayRef> {
```

Comment:
Should we also change this to just `Result<ArrayRef>`?

Reply:
Addressed by covering `functions-array` crate. Please see commit: 638c3a78edf2a85c696f233b40d85e83a383cc95